src/main/java/seedu/dictionote/model/AddressBook.java

```diff
@@ -15,6 +17,7 @@
 public class AddressBook implements ReadOnlyAddressBook {
 
     private final UniquePersonList persons;
+    private final UniqueNoteList note;
```

Comment:
May I ask why does the addressbook store note list? I remember there is a ReadOnlyNote class.